lib/net/modules/ethernet.toit

```diff
@@ -39,11 +41,20 @@ class Ethernet:
 
     if mac_chip == MAC_CHIP_ESP32:
       resource_group_ = ethernet_init_esp32_
+        mac_chip
         phy_chip
         phy_addr
         (phy_reset ? phy_reset.num : -1)
         mac_mdc.num
         mac_mdio.num
+    else if mac_chip == MAC_CHIP_OPENETH:
```

Comment:
Maybe share more of this code with the case above?

Reply:
done.